src/ServiceInsight/Framework/Behaviors/ComboBoxEditValuePatcherBehavior.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ServiceInsight.Framework.Behaviors
+{
+    using System.Windows.Data;
+    using DevExpress.Mvvm.UI.Interactivity;
+    using DevExpress.Xpf.Editors;
+    using ServiceInsight.ValueConverters;
+
+    public class ComboBoxEditValuePatcherBehavior : Behavior<ComboBoxEdit>
```

Comment:
I suppose this is the 'fix'? Should we put a comment there or somehow make it more obvious that this is not a generic behavior? Maybe a link to the bug report?